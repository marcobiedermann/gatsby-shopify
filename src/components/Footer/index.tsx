import React, { ReactNode } from 'react';
import * as styles from './style.module.css';

export interface FooterProps {
  children: ReactNode;
}

function Footer(props: FooterProps) {
  return <footer className={styles.footer} {...props} />;
}

export default Footer;
