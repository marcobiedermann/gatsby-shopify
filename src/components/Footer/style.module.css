.footer {
  background-color: #313739;
  color: #fff;
  padding-bottom: 6em;
  padding-top: 6em;
}
