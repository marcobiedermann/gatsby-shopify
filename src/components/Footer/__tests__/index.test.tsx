import { render } from '@testing-library/react';
import React from 'react';
import Footer from '..';

describe('footer component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Footer>Footer</Footer>);

    expect(asFragment()).toMatchSnapshot();
  });
});
