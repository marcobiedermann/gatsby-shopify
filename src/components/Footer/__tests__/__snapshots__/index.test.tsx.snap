// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer component should render correctly 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    Footer
  </footer>
</DocumentFragment>
`;
